src/main/webapp/app/modules/provider/all-records.tsx

```diff
@@ -21,6 +21,7 @@ const mapUrl = 'https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geomet
 const GRID_VIEW = 'GRID';
 const LIST_VIEW = 'LIST';
 const INACTIVE_COLOR = '#9aaabb';
+const MAP_OBJECT_KEY = '__SECRET_MAP_DO_NOT_USE_OR_YOU_WILL_BE_FIRED';
```

Comment:
Our map is a wrapper of the google map object. This is key to that wrapped object that is needed to use the transit layer.

Reply:
we can re-use the map constant from:
import { MAP } from 'react-google-maps/lib/constants